js/main.js

```diff
@@ -22,11 +22,87 @@ const getRandomFractionalNumber = function (min, max, symbolsAfterPoint) {
 
   let randomNumber = (getRandomNumber(min, max)).toFixed(symbolsAfterPoint);
   // If the number is fractional and ends in 0
-  if(randomNumber.slice(-1) === '0' && ((randomNumber ^ 0) === randomNumber) === false) {
+  if(randomNumber.slice(-1) === '0' && randomNumber % 1 !== 0) {
     randomNumber = randomNumber.substring(0, randomNumber.length - 1) + getRandomWholeNumber(1, 9) + randomNumber.substring(1 + randomNumber.length - 1);
   }
-  return Number(randomNumber);
+  return +randomNumber;
 }
 
-getRandomWholeNumber(0, 1);
-getRandomFractionalNumber(0, 2, 1);
+const TYPES = [
+  'palace',
+  'flat',
+  'house',
+  'bungalow',
+];
+
+const TIMES = [
+  '12:00',
+  '13:00',
+  '14:00',
+];
+
+const FEATURES = [
```

Comment:
Давай именовать более конкретно, мало ли о каких FEATURES речь
```suggestion
const ROOM_FEATURES = [
```
Аналогично с другими константами